jobs/kpi-forecasting/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.7
+MAINTAINER REPLACE ME <replaceme@mozilla.com>
+
+# https://github.com/mozilla-services/Dockerflow/blob/master/docs/building-container.md
+ARG USER_ID="10001"
+ARG GROUP_ID="app"
+ARG HOME="/app"
+
+ENV HOME=${HOME}
+RUN groupadd --gid ${USER_ID} ${GROUP_ID} && \
+    useradd --create-home --uid ${USER_ID} --gid ${GROUP_ID} --home-dir ${HOME} ${GROUP_ID}
+
+WORKDIR ${HOME}
+
+RUN apt install gcc
+RUN apt install g++
+RUN apt install gcloud
+
+RUN pip install --upgrade pip
+
+RUN pip install pystan==2.19.1.1
+RUN python3 -m pip install prophet --no-cache-dir
```

Comment:
I had originally tried it that way; but prophet and pystan especially are extremely particular about installations and this was the only way I could get it to consistently work =/

Reply:
We struggled a bunch with getting a working prophet environment in CI 2 years ago (see https://github.com/mozilla/forecasting/pull/4) but hopefully the situation has improved since then and the particulars of that struggle are no longer relevant.